tests/test_cases/test_package/test_package.py

```diff
@@ -41,6 +41,31 @@ async def test_stringification(dut):
     assert str(pkg2.eleven_int) == "IntegerObject(cocotb_package_pkg_2::eleven_int)"
 
 
+@cocotb.test(expect_fail=cocotb.SIM_NAME.lower().startswith("verilator"))
+async def test_long_parameter(dut):
+    """
+    Test package parameter access
+
+    # On verilator:
+    # 0.00ns ERROR    VPI error
+    # 0.00ns ERROR    vl_check_format: Unsupported format (vpiIntVal) for cocotb_package_pkg_1.long_param
+    # 0.00ns ERROR    Failed to initialize test test_long_parameter
+    """
+
+    pkg1 = cocotb.packages.cocotb_package_pkg_1
+    assert pkg1.five_int.value == 5
+    assert str(pkg1.five_int) == "IntegerObject(cocotb_package_pkg_1::five_int)"
+
+    assert pkg1.long_param.value.integer == int("5a89901af1", 16)
+    assert str(pkg1.long_param) == "LogicObject(cocotb_package_pkg_1::long_param)"
```

Comment:
`long_param` should be an integer (and Icarus and Verilator seem to think it is).

Reply:
Could we get __int__ defined on logic array so int(pkg1.long_param.value) would work for both interpretations?

Also, I had expect_fail set for verilator, am I doing something wrong there?